Client/Ecosia/UI/PageAction/PageActionMenu.swift

```diff
@@ -208,11 +211,49 @@ extension PageActionMenu: NotificationThemeable {
 // MARK: Cell
 
 class PageActionMenuCell: UITableViewCell {
+    private weak var badge: UIView?
+    private weak var badgeLabel: UILabel?
+    
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    func isNew(_ isNew: Bool) {
+        if isNew {
+            if badge == nil {
+                let badge = UIView()
+                badge.translatesAutoresizingMaskIntoConstraints = false
+                badge.isUserInteractionEnabled = false
+                badge.layer.cornerRadius = 10
+                contentView.addSubview(badge)
+                
+                let badgeLabel = UILabel()
+                badgeLabel.translatesAutoresizingMaskIntoConstraints = false
+                badgeLabel.font = .systemFont(ofSize: 12, weight: .semibold)
+                badgeLabel.text = .localized(.new)
+                
```

Comment:
```suggestion
                badgeLabel.adjustsFontForContentSizeCategory = true
```
to activate dynamic sizing :)